closure/goog/i18n/numberformat.js

```diff
@@ -580,6 +588,140 @@ goog.i18n.NumberFormat.prototype.roundNumber_ = function(number) {
 
 
 /**
+ * Formats when the length of the repeating digits is at least
+ * one digit. In the number grouping concept, anything left of the decimal
+ * place is followed by non-repeating digits and then repeating digits. If the
+ * pattern is #,##,###, then we first (from the left of the decimal place) have
+ * a non-repeating digit of size 3 followed by repeating digits of size 2
+ * separated by a thousand separator. If the length of the digits are six or
+ * more, there may be repeating digits required. For example, the value of
+ * 12345678 would format as 1,23,45,678 where the repeating digit is length 2.
+ *
+ * @param {Array<string>} parts The original parts variable that built in
+ *  goog.i18n.NumberFormat.prototype.subformatFixed_
+ * @param {number} zeroCode The original zeroCode variable from
+ *  goog.i18n.NumberFormat.prototype.subformatFixed_
+ * @param {string} intPart The original intPart variable from
+ *  goog.i18n.NumberFormat.prototype.subformatFixed_
+ * @param {Array<number>} groupingArray The array of numbers to determine the
+ *  grouping of repeated and non-repeated digits
+ * @param {number} repeatedDigitLen The length of the repeated digits left of
+ *  the non-repeating digits left of the decimal
+ * @return {!Array<!string>} Returns the resulting parts variable containing
```

Comment:
Sorry, technically `!string` is correct, but string a non-nullable type by default. You can just omit it here for consistency throughout the file.